telethon/network/mtprotosender.py

```diff
@@ -357,14 +358,15 @@ async def _reconnect(self, last_error):
         self._state.reset()
 
         retries = self._retries if self._auto_reconnect else 0
-        for attempt in retry_range(retries):
+        
+        attempt = 0
+        for attempt in retry_range(retries, False):
             try:
                 await self._connect()
             except (IOError, asyncio.TimeoutError) as e:
                 last_error = e
                 self._log.info('Failed reconnection attempt %d with %s',
-                               attempt, e.__class__.__name__)
-
+                                attempt, e.__class__.__name__)
```

Comment:
Could you get revert this change? Previously it was properly aligned with `(`, but it is no longer aligned (it's also not relevant to this PR).